middleware/jwt.go

```diff
@@ -15,22 +15,76 @@ import (
 	"errors"
 	"fmt"
 	"net/http"
+	"os"
+	"strconv"
 	"strings"
+	"time"
 
 	"github.com/auth0/go-jwt-middleware"
 	jwt "github.com/dgrijalva/jwt-go"
 	"github.com/gorilla/context"
+	"github.com/heketi/heketi/pkg/utils"
 )
 
+var (
+	logger          = utils.NewLogger("[jwt]", utils.LEVEL_DEBUG)
+	iatLeeway int64 = 120
+)
+
+func init() {
+	if leeway, exists := os.LookupEnv("HEKETI_JWT_IAT_LEEWAY_SECONDS"); exists {
```

Comment:
I have a question, what happens if we set ```HEKETI_JWT_IAT_LEEWAY_SECONDS``` as a negative value.

Reply:
Handled now and added a test.